tests/test_core.py

```diff
@@ -8852,7 +8854,6 @@ def test_safe_stack_dylink(self):
       }
     ''', ['abort(stack overflow)', '__handle_stack_overflow'], assert_returncode=None)
 
-  @also_with_standalone_wasm
   def test_undefined_main(self):
```

Comment:
do we now error on this case? perhaps worth adding a test for that, as otherwise it seems we lost coverage without covering the new behavior.

Reply:
There was a lot of overlap with  `test_auto_detect_main` below, so I will merge them.